src/jiti.ts

```diff
@@ -235,7 +236,7 @@ export default function createJITI (_filename: string, opts: JITIOptions = {}, p
     )
 
     if (needsTranspile) {
-      debug('[transpile]', filename)
+      debug(`[transpile${isNativeModule ? ' native' : ''}]`, filename)
```

Comment:
CJS is also native :) 

```suggestion
      debug(`[transpile${isNativeModule ? ' esm' : ''}]`, filename)
```

Reply:
module is better

Reply:
esm also stands for "EcmaScript Modules". However, i understand it can also be misleading considering non standard ESM syntax but then I would call that "bad esm" :D